ethereum/core/src/main/java/tech/pegasys/teku/core/signatures/SigningRootUtil.java

```diff
@@ -41,14 +42,23 @@ public Bytes signingRootForRandaoReveal(final UInt64 epoch, final ForkInfo forkI
   }
 
   public Bytes signingRootForSignBlock(final BeaconBlock block, final ForkInfo forkInfo) {
-    final SpecVersion specVersion = spec.atSlot(block.getSlot());
+    return spec.computeSigningRoot(block, getDomainForSignBlock(block.getSlot(), forkInfo));
+  }
+
+  public Bytes signingRootForSignBlock(
```

Comment:
Maybe call this `signingRootForSignBlockHeader`?

Reply:
done.